internal/zoom/cover.out

```diff
@@ -0,0 +1,18 @@
+mode: set
```

Comment:
Test coverage artifacts shouldn't be in the repo

Reply:
fixed